charts/airflow/examples/google-gke/custom-values.yaml

```diff
@@ -315,6 +315,15 @@ dags:
     ##
     sshSecretKey: id_rsa
 
+    ## NOTE:
+    ## - when `workers.autoscaling` is true, YOU MUST SPECIFY a resource request
+    ##
+    ## EXAMPLE:
+    resources:
```

Comment:
@david30907d Can you format this docstring in the standard way:

```
## resource requests/limits for the git-sync container
##
## WARNING:
## - you MUST SPECIFY a resource request for gitSync if using `workers.autoscaling`
##
resources:
  ...
```

Reply:
done 😄